packages/demo/src/index.js

```diff
@@ -162,20 +162,28 @@ async function main() {
   }
 
   async function renderExampleButtons() {
-    const buttonContainer = document.querySelector("#example-streams");
-    ExampleStreams.forEach(async (exampleStream) => {
-
-      const manifestType = await getManifestType(exampleStream.url);
-      const nativelySupportedManifestType = canPlayManifestType(manifestType);
-      const isIOS = /iPhone|iPod/.test(navigator.userAgent);
-      const isIPadOS = /iPad/.test(navigator.userAgent) || (/Macintosh/.test(navigator.userAgent) && "ontouchstart" in document.documentElement);
-      if ((isIOS || isIPadOS) && !nativelySupportedManifestType) return false;
+    const buttonContainer = document.querySelector('#example-streams');
+
+    const playableStreams = await Promise.all(
+      ExampleStreams.map(async (exampleStream) => {
```

Comment:
Is it really necessary to do a `map` iteration here as you do no real changes to the objects?
I assume it is done due to `.foreach` not working with `await` / `async`.
Maybe read into this https://zellwk.com/blog/async-await-in-loops/

Reply:
Ah 👍